default.nix

```diff
@@ -1,9 +1,6 @@
 { lib, pkgs, ...}:
 
-pkgs.rustPlatform.buildRustPackage rec {
-  pname = "magmawm";
-  version = "main";
-
+pkgs.rustPlatform.buildRustPackage {
```

Comment:
still need to declare pname and version
```nix
 lib, pkgs, ...}: let
  pname = "magmawm";
  version = "main";
in pkgs.rustPlatform.buildRustPackage {
  inherit pname version;
```

Doing it this way means you can still use `meta.mainProgram = pname`